Gaijinizer patch/Map099.txt

```diff
@@ -225,9 +237,9 @@ Young leader .........」#MTLed
 
 若干迂回し、ヘリの墜落現場に到着する辺りです。
 > CONTEXT: Map099/events/1/pages/0/150/Dialogue/J_People1
-House………. Anomalous growth of plants in the
-square. A little detour is about to arrive at the
-crash site of the helicopter.#MTLed
+No... We've detected anomalous growth of plants
+in the clearing, he's now circling around and is
+about to arrive at the helicopter crash site.#CheckLater
```

Comment:
Not sure about your #CheckLater, but if you aren't changing this change the comma after "clearing" to a period.